app/controllers/home_controller.rb

```diff
@@ -6,7 +6,7 @@ class HomeController < ApplicationController
   before_action :set_instance_presenter
 
   def index
-    expires_in 0, public: true unless user_signed_in?
```

Comment:
There are more places where `expires_in 0` appears, should they be updated too?

Reply:
Oh, yeah, probably. Going to have a look.

Reply:
Done, although with a lower `stale_if_error` for accounts and tags.